script/bootstrap/bitcoin_genesis.ts

```
@@ -0,0 +1,607 @@
+        console.error('Error fetching transactions:', error);
+        throw error;
+      }
+    }
+
+    return allTxs;
+  }
+
+  private async isValidatorRegistered(validatorAddr: string): Promise<boolean> {
+    try {
+      const validatorInfo = await this.bootstrapContract.validators(validatorAddr);
+      return validatorInfo && validatorInfo.name && validatorInfo.name.length > 0;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)